website/src/components/Changelog/GUI.tsx

```diff
@@ -11,6 +11,19 @@ export default function GUI({ title }: { title: string }) {
 
   return (
     <Entries href={href} arches={arches} title={title}>
+      {/*<Entry version="1.1.6" date={new Date("2024-07-10")}>
+        <ul className="list-disc space-y-2 pl-4 mb-4">
+          <li className="pl-2">
+            Fixes{" "}
+            <Link
+              href="https://github.com/firezone/firezone/pull/5827"
+              className="text-accent-500 underline hover:no-underline"
+            >
+              a bug where the Windows Client doesn't deactivate DNS control when you sign out.
```

Comment:
Forgot to mention it in the issue - No, this is an oddity of using NRPT. I tried to reproduce it on Linux, but systemd automatically reverts DNS when connlib stops, so we don't have this problem on Linux.

And since the bug is in the IPC service it shouldn't affect the headless Client, and the headless Client doesn't have a "running but signed out" state anyway